examples/buildless/now.json

```diff
@@ -0,0 +1,21 @@
+{
+  "builds": [
```

Comment:
The [`builds`](https://zeit.co/docs/configuration#project/builds) property is deprecated.

In fact, examples should not be using `now.json` at all, they should embrace [zero config](https://zeit.co/blog/zero-config).

Reply:
Totally makes sense, I will update that based on your feedback 👍.